whisper-fine-tuning-sprint/README.md

```diff
@@ -0,0 +1,635 @@
+# Whisper fine-tuning sprint 🤗
+
+Welcome to the Whisper fine-tuning sprint 🎙️ !
+
+The goal of this event is to build **robust**, **real-world** speech recognition (ASR) systems in as many languages as possible 🌏🌍🌎.
+If necessary and available, free access to a V100s 32 GB GPU will kindly be provided by the [PUT CLOUD PROVIDER HERE](https://PUTCLOUDPROVIDERHERE.COM) 🚀.
+This document summarizes all the relevant information required for the speech community event 📋.
+
+To sign-up, please see [this forum post](https://LINK_ACTUAL_FORUM) 🤗. Please make sure to:
```

Comment:
I don't think the forum post should be required for people here. Going through this document should be everything they need, otherwise, we'll lose people jumping from one link to another. This page has everything important I think